### ISSUE TITLE
Deep links show other cards if on different networks

### ISSUE THREAD

**Issue by ldanielswakman** [RadiCards/radi.cards#25]
If you have your network set to Ropsten and you go to a card deep link that is on main net, it shows you another card on Ropsten (if the ID exists at all). No clear solution for now, but needs to be handled gracefully

**Comment by SoIidarity** [comment]
@ldanielswakman what do you think about putting a banner on the top of the website that says if you are not connected to main net? On all pages it'll show you something like "warning! currently connected to ropsten! please change to the main net to see the official Radicards"

**Comment by ldanielswakman** [comment]
Yeah that's a good idea; actually it depends on what kind of role we want Ropsten network to have. If it is really just a testing network for development purposes, and we don't want people to mint too many cards on it — then I would say we can even be clearer and have a popup on top the page making it clear that you're currently viewing Ropsten, and that the app is intended to work on Mainnet... @Lililashka what do you think?
